index.html

```diff
@@ -3,7 +3,7 @@
         <title>My website</title>
```

Comment:
I changed it to PHP this morning but I guess I didn't refresh it in Github before sending it your way. 

As far as notes, I started off trying to get some in but I also wanted to follow along. I decided to follow along but in the future I'm going to focus on notes and then practice on my own. Code Academy was a great tool for me because I can look at the directions as many times as I need to. 

Thanks for your help!